summoner-tui/src/Summoner/Tui/Field.hs

```diff
@@ -39,6 +39,7 @@ strField t _ = FormFieldState
     , formFields = []
     , formFieldRenderHelper = renderString
     , formFieldConcat = vBox
+    , formFieldUpdate = flip const
```

Comment:
I'm still puzzled by the meaning of this function, but what can we do...

Reply:
Me too. But we don't use this functionality (I hope). It is not completely clear if it should be `const` or `flip const`. Do you have an idea here?

While in the first case it doesn't matter (as the type there is `() -> () -> ()`, but in the second change we work with `Bool`s 🤔

Reply:
Yeah, I'm very confused. I tried to investigate but failed. Let's keep the current version then 👍🏻